src/eventer/eventer_impl_private.h

```diff
@@ -144,6 +152,20 @@ void eventer_impl_init_globals(void);
 void eventer_dispatch_recurrent(void);
 void eventer_dispatch_timed(struct timeval *next);
 void eventer_mark_callback_time(void);
+void eventer_set_this_event(eventer_t e);
+void eventer_callback_prep(eventer_t, int, void *, struct timeval *);
+void eventer_callback_cleanup(eventer_t, int);
+
+static inline int RUN_CALLBACK(eventer_t e, int m, void *c, struct timeval *n) {
```

Comment:
I thought this was a macro when reading higher up.. maybe `eventer_run_callback` so it's clear it's a function?